time/src/macros.rs

```diff
@@ -65,7 +65,6 @@ pub use time_macros::datetime;
 /// );
 /// # Ok::<_, time::Error>(())
 /// ```
-///
```

Comment:
You can remove these changes. I know `rustfmt` is failing, but it's a bug upstream and not something that needs to be changed here.